smaug/python/create_model_example.py

```diff
@@ -1,99 +1,87 @@
+
+Note that all the tensors and operators must be created within
+a Graph context. The target backend is also specified via the
+Graph context. See the below for examples. The user can get a
+summary of the graph by calling print_summary(). To serialize
+the graph into a pb file, call write_graph().
+"""
 
 import numpy as np
-from smaug.core import types_pb2
-from smaug.python.graph import Graph
-from smaug.python.tensor import Tensor
-from smaug.python.ops import array_ops, math_ops, data_op, nn_ops
+import smaug as sg
 
 def create_residual_model():
-  with Graph(name="residual_graph", backend="SMV") as graph:
+  with sg.Graph(name="residual_graph", backend="SMV") as graph:
     # Tensors and kernels are initialized as NCHW layout.
-    input_tensor = Tensor(
+    input_tensor = sg.Tensor(
         tensor_data=np.random.rand(1, 1, 28, 28).astype(np.float16))
-    filter_tensor0 = Tensor(
+    filter_tensor0 = sg.Tensor(
         tensor_data=np.random.rand(64, 1, 3, 3).astype(np.float16))
-    filter_tensor1 = Tensor(
+    filter_tensor1 = sg.Tensor(
         tensor_data=np.random.rand(64, 1, 3, 3).astype(np.float16))
-    filter_tensor2 = Tensor(
+    filter_tensor2 = sg.Tensor(
         tensor_data=np.random.rand(64, 64, 3, 3).astype(np.float16))
-    bn_mean_tensor = Tensor(
-        data_layout=types_pb2.NC,
-        tensor_data=np.random.rand(1, 64).astype(np.float16))
-    bn_var_tensor = Tensor(
-        data_layout=types_pb2.NC,
-        tensor_data=np.random.rand(1, 64).astype(np.float16))
-    bn_gamma_tensor = Tensor(
-        data_layout=types_pb2.NC,
-        tensor_data=np.random.rand(1, 64).astype(np.float16))
-    bn_beta_tensor = Tensor(
-        data_layout=types_pb2.NC,
-        tensor_data=np.random.rand(1, 64).astype(np.float16))
-
-    act = data_op.input_data(input_tensor)
-    x = nn_ops.convolution(act, filter_tensor0, stride=[1, 1], padding="same")
-    out = nn_ops.convolution(act, filter_tensor1, stride=[1, 1], padding="same")
-    out = nn_ops.batch_norm(
+    bn_mean_tensor = sg.Tensor(
+        data_layout=sg.NC, tensor_data=np.random.rand(1, 64).astype(np.float16))
+    bn_var_tensor = sg.Tensor(
+        data_layout=sg.NC, tensor_data=np.random.rand(1, 64).astype(np.float16))
+    bn_gamma_tensor = sg.Tensor(
+        data_layout=sg.NC, tensor_data=np.random.rand(1, 64).astype(np.float16))
+    bn_beta_tensor = sg.Tensor(
+        data_layout=sg.NC, tensor_data=np.random.rand(1, 64).astype(np.float16))
+
+    act = sg.input_data(input_tensor)
+    x = sg.nn.convolution(act, filter_tensor0, stride=[1, 1], padding="same")
+    out = sg.nn.convolution(act, filter_tensor1, stride=[1, 1], padding="same")
+    out = sg.nn.batch_norm(
         out, bn_mean_tensor, bn_var_tensor, bn_gamma_tensor, bn_beta_tensor,
-        activation=types_pb2.ReLU)
-    out = nn_ops.convolution(out, filter_tensor2, stride=[1, 1], padding="same")
-    out = math_ops.add(x, out)
+        activation=sg.ReLU)
```

Comment:
`sg.ReLU` could be problematic because we also have a dedicated relu operator. How would a user distinguish that from this?

Reply:
I think a better way to fix this is making the argument as string type like `"relu"` instead of directly using an internal type.

Reply:
That sounds good to me.